incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -17,13 +17,14 @@ package kubectl
 
 import (
 	"fmt"
+	"os"
 	"sort"
 	"strconv"
 	"strings"
 
-	"github.com/spf13/cobra"
-
 	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Usually we have up to three "groups" of imports:

- Standard library ("os", "strings", etc)
- Packages from other projects ("cobra", "metav1", etc)
- Packages from this project ("api")

I won't hold this PR up for this change, but I'll probably change it back myself.